site/schemas/session.schema.ts

```diff
@@ -11,19 +11,23 @@ export const sessionSchema = z
         given_name: z.string().optional(),
         family_name: z.string().optional(),
         "custom:orgId": z.string().uuid(),
+        "custom:operatorOrgId": z.string().uuid().optional(),
         "cognito:groups": z.array(z.nativeEnum(UserGroups)).optional(),
     })
     .transform((item) => {
         const isSystemAdmin = item["cognito:groups"]?.includes(UserGroups.systemAdmins) ?? false;
         const isOrgAdmin = item["cognito:groups"]?.includes(UserGroups.orgAdmins) ?? false;
         const isOrgPublisher = item["cognito:groups"]?.includes(UserGroups.orgPublishers) ?? false;
         const isOrgStaff = item["cognito:groups"]?.includes(UserGroups.orgStaff) ?? false;
+        const isOperatorUser = item["cognito:groups"]?.includes(UserGroups.operators) ?? false;
 
         return {
```

Comment:
Might need some clarification on AC3 for this ticket which is about setting the datasource for operator users. 

At the moment, the datasource will be whatever the LTA org has set - if we follow the ACs then we will need some logic on line 45 to set the datasource for operator users

Reply:
done